be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -28,7 +28,7 @@ SinkBuffer::SinkBuffer(FragmentContext* fragment_ctx, const std::vector<TPlanFra
                        bool is_dest_merge)
         : _fragment_ctx(fragment_ctx),
           _mem_tracker(fragment_ctx->runtime_state()->instance_mem_tracker()),
-          _brpc_timeout_ms(std::min(3600, fragment_ctx->runtime_state()->query_options().query_timeout) * 1000),
+          _brpc_timeout_ms(fragment_ctx->runtime_state()->query_options().query_timeout * 1000),
```

Comment:
what about query_timeout = 0

Reply:
"Getting analyzing error. Detail message: query_timeout must be equal or greater than 1." There is a check in analyze phase.